frontend/src/common-components/layout/CmschContainer.tsx

```diff
@@ -9,8 +9,8 @@ export const CmschContainer = ({ children }: Props) => (
     px="4"
     py="4"
     mx="auto"
-    maxWidth={['100%', '48rem', '48rem', '64rem']}
-    borderRadius={[0, 10]}
+    maxWidth={['100%', '48rem', '48rem', '48rem']}
```

Comment:
Nem lesz túl szűk így a látványtér?

Reply:
A content így beszűkül a nagyobb viewportokon is

Reply:
de max meglátjuk

Reply:
ja a háttér miatt van értelme, a contentünk úgy is mobile first